reverse-proxy/traefik/docker-compose.traefik.yml

```diff
@@ -1,8 +1,95 @@
-version: "3.3"
+version: '3.3'
+
+networks:
+  live_network:
+    external: true
+  internal_network:
+    external: false
+
 services:
+  traefik:
+    container_name: "traefik"
+    image: "traefik:v2.9"
+
+    networks:
+      - "live_network"
+      - "internal_network"
+
+    restart: unless-stopped
+
+    command:
+      - "--entrypoints.web.address=:80"
+      - "--entrypoints.websecure.address=:443"
+      - "--providers.docker=true"
+      - "--certificatesresolvers.le.acme.email=example@example.com"  # Add your email address here
+      - "--certificatesresolvers.le.acme.storage=/data/acme.json"
+      - "--certificatesresolvers.le.acme.tlschallenge=true"
+
+    ports:
+      - "80:80"
+      - "443:443"
+
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+      - "./data:/data"
+
+    labels:
+      - "traefik.enable=false"
+
   rallly:
+    container_name: "rallly"
+    image: lukevella/rallly:latest
+    restart: unless-stopped
+
+    networks:
+      - "live_network"
+      - "internal_network"
+
+    depends_on:
+      rallly-db:
+        condition: service_healthy
+
+    environment:
+      - DATABASE_URL=postgres://postgres:postgres@rallly-db:5432/db
+
+    env_file:
+      - ./config/rallly.env
```

Comment:
Would this need to point to `../config.env`?

Reply:
I added a `TODO` comment similar to the `Host` directives to instruct users to replace this with the path to their own env file.